src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1

```diff
@@ -295,6 +295,62 @@ Test-CreateAndGetSoftwareUpdateConfigurationWithAzureDynamicGroupsOnlyWithOutTag
                                        -StartTime $startTime `
                                        -ForUpdate
 
+$query1Scope = @(
+        "/subscriptions/cd45f23b-b832-4fa4-a434-1bf7e6f14a5a/resourceGroups/mms-wcus"
```

Comment:
This hard-coded subscription Id will cause problems in re-recording the test

Reply:
these tests need prep environements before they run. in order to re-record the test, the steps are documented on the top of the file.